dbeam-core/src/main/java/com/spotify/dbeam/options/DBeamPipelineOptions.java

```diff
@@ -38,6 +38,11 @@
 
   void setTable(String value);
 
+  @Description("A local file containing the SQL SELECT query.")
```

Comment:
I think the file reader we have would allow us to read the file from GCS. If so we should update this description.

Reply:
Addressed.

Reply:
Updated.